CONTRIBUTORS.rst

```diff
@@ -327,6 +327,7 @@ Contributors
 * Daniele Procida
 * Catherine Farman
 * Abdulmalik Abdulwahab
+* Thiago Costa de Souza
```

Comment:
Just a note for the future - no need to add this, we will do it when merged. Avoids conflicts. 

Leave this in the PR though for now. Thank you